test/langtools/tools/javac/4980495/static/Test.java

```diff
@@ -11,7 +11,7 @@
 
 public class Test {
 
-    public static void main(String argv[]) {
+    public static void meth() {
```

Comment:
not sure about this case but if for some tests the method doesn't need to be `static` we should remove the modifier

Reply:
Yes - in fact, at first I started doing that as well, but it turns out that doing this wholesale generates a bunch of test failures, and therefore requires a good bit more analysis to figure out which tests can handle it and which can't. The primary goal here is to remove misleading `public static void main(String[])` methods, not minimize in general. Perhaps more a general test minimization effort could be relegated to "phase 2", i.e., another bug?

Reply:
sure I'm OK with that